docs/app-containers.md

```diff
@@ -6,67 +6,53 @@ sidebar_label: App Containers
 
 Containers are responsible for managing your app state and linking your top-level navigator to the app environment. On Android, the app container uses the Linking API to handle the back button. The container can also be configured to persist your navigation state. On web, you'd use different containers than React Native.
 
-> Note: In v2 and earlier, the containers in React Navigation are automatically provided by the `create*Navigator` functions. As of v3, you are required to use the container directly. In v3 we also renamed `createNavigationContainer` to `createAppContainer`.
-
-A quick example of `createAppContainer`:
+A quick example of `NavigationContainer`:
 
 ```
-import { createAppContainer, createStackNavigator } from 'react-navigation';
-// you can also import from @react-navigation/native
-
-const AppNavigator = createStackNavigator(...);
-
-const AppContainer = createAppContainer(AppNavigator);
-
-// Now AppContainer is the main component for React to render
+import { NavigationContainer } from '@react-navigation/core';
+import { createStackNavigator } from '@react-navigation/stack';
+// you can also import NavigationContainer from @react-navigation/native
```

Comment:
I think we should import from `/native` in docs since its necessary to handle back button. The commend makes it seem like that it's the same.